doc/rst/technotes/gpu.rst

```diff
@@ -178,6 +185,12 @@ section; many of them will be addressed in upcoming editions.
 * Runtime checks such as bounds checks and nil-dereference checks are
   automatically disabled for CHPL_LOCALE_MODEL=gpu.
 
+* For AMD GPUs:
+
+    * Multiple locales are not supported
```

Comment:
Maybe "can only be used with local builds (i.e., ``CHPL_COMM=none``)" ?

Reply:
Sure I'll change it your wording.

I'm actually not 100% sure this isn't the case either (maybe there's some environment variables I need to tweak to get it to work). Just that I get an error when I try and run it on our CrayCS machine. See this comment for more details: https://github.com/Cray/chapel-private/issues/4477#issuecomment-1461074091